articles/multifactor-authentication/api/oob.md

```diff
@@ -1,14 +1,22 @@
 ---
-title: Associate an OOB Authenticator
-description: How to associate an OOB authenticator
+title: Associate an Out-of-Band Authenticator
+description: Configure your application so users can self-associate out-of-band (OOB) authenticators.
 ---
-# Associate an OOB Authenticator
 
-In this tutorial, we will show you how you can configure your Auth0 tenant to allow the self-association of out-of-band (OOB) authenticators.
+# Associate an Out-of-Band Authenticator
 
-## Step 1. Trigger the MFA Error and Use the MFA Token to Associate the New Authenticator
+In this tutorial, you'll learn how to configure your application so users can self-associate out-of-band (OOB) authenticators.
 
-Whenever a user begins the authorization process and they do not have an active authenticator associated with their account, they will trigger the following MFA response when calling the `/oauth/token` endpoint:
+## Before you start
+
```

Comment:
reuse content everywhere you can. I saw the same section at the articles/multifactor-authentication/api.md doc